drivers/at86rf2xx/at86rf2xx.c

```diff
@@ -70,18 +70,17 @@ void at86rf2xx_reset(at86rf2xx_t *dev)
     dev->netdev.flags = 0;
     /* set short and long address */
 #if CPUID_LEN
+    /* in case CPUID_LEN < 8, fill missing bytes with zeros */
+    memset(&(cpuid[CPUID_LEN]), 0, CPUID_LEN);
```

Comment:
Here and below: this call let's the buffer overflow in any case. Please use either `cpuid` or `&cpuid[0]` as a starting point.